src/sass/static/pages.scss

```diff
@@ -669,3 +669,68 @@ body #not_authenticated_financial ul.checked > li {
         }
     }
 }
+
+.otp-form-group {
```

Comment:
Wrapping all styles of this page in a page-specific selector, would prevent unwanted changes to other pages.